dockerfiles/jaeger/agent/Dockerfile

```
@@ -0,0 +1,18 @@
+# This Dockerfile tracks jaegertracing/jaeger-agent, but includes more debugging tools and runs as a
+# non-root user. It requires JAEGER_VERSION to be set as an argument to build.
+ARG JAEGER_VERSION
+FROM jaegertracing/jaeger-agent:$JAEGER_VERSION as base
+
+FROM alpine:3.11.5
+RUN apk update
+RUN apk add bash curl
```

```suggestion
RUN apk --no-cache bash curl
```

we don't need to distribute the apk cache in the image, the above suggestion does what you want without doing that.